HiP-UserStore.Model/Entity/User.cs

```diff
@@ -2,9 +2,26 @@
 {
     public class User : ContentBase
     {
+        public string Email { get; set; }
+
+        public string FirstName { get; set; }
+
+        public string LastName { get; set; }
+
+        public StudentDetails StudentDetails { get; set; }
+
         /// <summary>
         /// The path to the actual profile picture file.
         /// </summary>
         public string ProfilePicturePath { get; set; }
     }
+
+    public class StudentDetails
+    {
```

Comment:
I think it would be better to make a constructor which will get StudentDetailsArgs